streaming/src/main/scala/org/apache/spark/streaming/ui/AllBatchesTable.scala

```diff
@@ -26,7 +26,7 @@ private[ui] abstract class BatchTableBase(tableId: String) {
   protected def columns: Seq[Node] = {
     <th>Batch Time</th>
       <th>Input Size</th>
-      <th>Scheduling Delay</th>
+      <th>Streaming Scheduling Delay</th>
```

Comment:
This is the screenshot for the table:
![f2](https://cloud.githubusercontent.com/assets/1000778/7443650/ec4acd4e-f111-11e4-84ac-616d39ffc060.png)